Makefile

```diff
@@ -160,6 +160,16 @@ test-integration-race:
 	@go test -race -tags=$(TAGS) ./nodebuilder/tests
 .PHONY: test-integration-race
 
+## test-e2e: Run an end to end test via docker.
+test-e2e:
+	@if [ -z "$(test)" ]; then \
+		echo "ERROR: 'test' variable is required. Usage: make test-e2e test=TestE2EMsgPayForBlob"; \
+		exit 1; \
+	fi
+	@echo "--> Running: TestCelestiaTestSuite/$(test)"
+	cd e2e && go test -v -run ^TestCelestiaTestSuite/$(test)$$ ./...
```

Comment:
`go test -v -run ^TestCelestiaTestSuite/$(test)$$ ./e2e` should work as well.

BTW, why do we need to specify test all the time? Why not run all the test by default unless specific test(s?) is specified? Just curious.